lang/gcc10-bootstrap/Portfile

```diff
@@ -250,16 +239,50 @@ build.target        bootstrap-lean
 # They differ when universal due to gcc multilib being messy non-universal
 destroot.delete_la_files yes
 
+post-destroot {
+    foreach f [glob -nocomplain -d ${destroot}${prefix}/bin c++ g++ gcc gcc-ar gcc-nm gcc-ranlib] {
+        file delete $f
+    }
+
+    set cc [open "${destroot}${prefix}/bin/cc" w 0755]
+    puts ${cc} "#!/bin/sh"
+    puts ${cc} "GCC_TARGET=\"\$\{GCC_TARGET:-$merger_host(${build_arch})\}\""
+    puts ${cc} "PROG=\$(basename \"\$0\")"
```

Comment:
From memory `$(basename $0)` won’t work for via symlinks, instead just add the script for each binary